src/components/CustomMarker/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import MarkerView from 'components/common/MarkerView';
+import { number, func, string, bool } from 'prop-types';
+import useTopicIcon from 'hooks/useTopicIcon';
+import { Marker } from 'react-native-maps';
+
+const CustomMarker = ({ id, lat, lng, topicId, onPress, selected }) => {
```

Comment:
Mabey you can swap the names, CustomMarker is moore of the name that the documentation tells you to name it. But in reality it is the Marker of your App. What you can do to solve name conflicts is to rename the react-native-maps marker like this. import { Marker as MapMarker } from 'react-native-maps';

Reply:
Oops. Actually, it is TargetMarker, since Peter commented the same in the last pr. It was a matter with the git rebase. I'll delete this file. Thanks!